how-to/how-to-clear-browser-cache-and-delete-cookies.md

```diff
@@ -0,0 +1,38 @@
+---
+title: How to Clear Browser Cache and Delete Telerik Platform Cookies
+description: Step by step tutorial for clearing the browser cache and deleting the Telerik Platform cookies.
+type: how-to
+page_title: How to Delete Browser Cache and Remove Telerik Platform Cookies
+slug: clear-browser-cache
+position:
+tags: other
+teampulseid: 
+ticketid: 
+pitsid: 
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Third-party product</td>	
+  </tr>
+</table>
+
+## Description
+This article provides the general steps for clearing the browser cache and deleting the Telerik Platform cookies.
+
+## Solution
+In all browsers you need to:
+
+1. Log out from [platform.telerik.com](https://platform.telerik.com) and [telerik.com](https://www.telerik.com).
+2. Clear the cache and your browser cookies for `app.icenium.com` and `platform.telerik.com`. 
+> **IMPORTANT:** In case you use Google Chrome, make sure to select clear cookies from beginning of time.
+
+## Notes
+To verify if the issues are resolved you should log in [platform.telerik.com](https://platform.telerik.com). 
```

Comment:
... you should log in **to** [platform.telerik.com](https://platform.telerik.com) **and see if the issue still reproduces**.